host/4/node-build.yml

```diff
@@ -30,6 +32,12 @@ steps:
     continueOnError: false
     env:
       pswd: $(dockerPassword)
+  
+  - bash: |
+      ./host/generate-composite.sh -4 node
```

Comment:
How about adding PublishArtifact task for pushing the generated Dockerfile for debugging?

Reply:
This is a great idea. I am just not sure how we would publish that artifact or where it should go. The build pipelines should tell us if any images failed building.  But, it would be nice to have slightly higher traceability if there was some way to surface that artifact on the pipeline run of build that happens.

Reply:
This might help. https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-build-artifacts?view=azure-devops

Reply:
This is great Tsuyoshi. I think that I can and should use this to produce artifacts during build pipelines for each language. I have captured your comment in an issue on the repo here. 
https://github.com/Azure/azure-functions-docker/issues/560

Feels like something that should be addressed in a follow up pr but I agree that it is useful